vshard/storage/init.lua

```diff
@@ -1085,6 +1085,22 @@ local function rebalancing_is_in_progress()
     return f ~= nil and f:status() ~= 'dead'
 end
 
+local function recovery_bucket_stat(bid)
+    local ok, err = check_is_master()
+    if not ok then
+        return nil, err
+    end
+    local b = box.space._bucket:get{bid}
+    if b == nil then
+        return
```

Comment:
This function deliberately treats absence of the bucket as not an error. Because I just don't need the error. The only possible error is that the function is called not on a master instance.